R/TADARefTables.R

```diff
@@ -1,3 +1,40 @@
+#' ATTAINS Parameter and Use Name by Organization Reference Key
+#'
+#' Function downloads and returns the newest available ATTAINS domain values
+#' reference dataframe summarized by parameter, use and organization.
```

Comment:
Add clarifying language about parameters (listed as cause by org in previous assessments)